win32/tests/threadpool_win32_int/threadpool_win32_int.c

```diff
@@ -119,12 +124,15 @@ BEGIN_TEST_SUITE(threadpool_win32_inttests)
 
 TEST_SUITE_INITIALIZE(suite_init)
 {
+    xlogging_set_log_function(NULL);
```

Comment:
>xlogging_set_log_function [](start = 4, length = 25)

why disable logging?

Reply:
Good question.

So here's 2 reasons why it was disabled
1) CTEST would have problems ingesting all the output (or so I am hypothesizeing) - once this was taken away, CTest was glorious again
2) it is a good idea in general when working with threading when wanting to prove that threading "works". Reason is logging takes a lock... and that might interfere with the threading, and it is better without it.

---
In reply to: [462453205](https://github.com/Azure/azure-c-pal/pull/21#discussion_r462453205) [](ancestors = 462453205)

Reply:
What about asserts? They use xlogging...

---
In reply to: [463141535](https://github.com/Azure/azure-c-pal/pull/21#discussion_r463141535) [](ancestors = 463141535,462453205)